typescript/infra/scripts/check/check-utils.ts

```
@@ -278,7 +207,11 @@ const getGovernor = async (
           chain ?? fork
         } is non evm and it not compatible with warp checker tooling`,
       );
-      return;
+      throw Error(
```

Any concerns with this function throwing @paulbalaji, It is called during chain deployments

---

na let it throw, don't think it's a problem